docs/timelines/timeline-options.md

```diff
@@ -0,0 +1,102 @@
+title: Timeline Options
+sidebar_label: Timeline options
+---
+
+Timelines have a number of options which set how they operate. 
+Press \<Options\> (or the \[Options\] softkey on the top level menu) then the **select** button for the timeline you wish to edit.
+
+--- 
+
+## Handle Tab
+
+All options are the same as for a cue, see [Options](../cues/playback-options.md#handle-tab).
+
+--- 
+
+## Times Tab
+
+![Timeline Options: Times](/docs/images/Timeline-Options-Times.png)
+
+These options let you set the end time and start time of the timeline. These times set the loop points when you are using the 
+[Loop function](../timelines/timeline-options.md#timecode-tab) or can be used to set limits on the timeline so you can be sure nothing will happen if you get sent some wayward timecode.
+
+--- 
+
+## Fader Tab
+
+![Timeline Options: Fader](/docs/images/Timeline-Options-Fader.png)
+
+Sets what happens to an internal timecode source when the playback fader containing the timeline is raised or zeroed. If an external
+timecode source is being used then this option has no effect.
+
+Option              | Action
+--------------------|---
+Fader Raised Action | **No Action**: The internal timecode source is not changed (default).<br>Play: The internal timecode source is started.
+Fader Zero Action   | **No Action**: The internal timecode source is not changed (default).<br>Stop: The internal timecode source is stopped. <br>Pause: The internal timecode source is paused at the current time.
+
+
+--- 
+
+## Release Tab
+
+![Timeline Options: Release](/docs/images/Timeline-Options-Release.png)
+
+Sets what happens to LTP attributes in playbacks which are killed by the timeline. This can be useful to make the output
+more predictable when you are skipping around the timeline, otherwise you can get confusing LTP values
+persisting from previous playbacks.
+
+Option              | Action
+--------------------|---
+Override Playback Release | **Off**: The playback's own release settings are used.<br>On: The Timeline-Global release settings override the playback settings.
+Release Playbacks to Home | Off: LTP attributes are left when when killing playbacks.<br>**On**: Attributes will return to home if no previous LTP values are set.
+Timeline-Global Release Mask | Sets the Global Release Mask for playbacks triggered by the timeline (default **no attributes released**).
+Timeline-Global Release Time | Sets the Global Release Time for playbacks triggered by the timeline (default **2 seconds**).
+
+--- 
+
+## Timecode Tab
+
+![Timeline Options: Timecode](/docs/images/Timeline-Options-Timecode.png)
+
+Option              | Action
+--------------------|---
+Autoload In Range | Off: Timeline must be fired manually. <br>**On:** Timeline will be fired automatically when timecode comes within the Start/End time range.
+Kill At Zero | **Off:** Timeline will remain active when the timeline fader is zeroed. <br>On: Timeline will be killed when the timeline fader is zeroed. 
```

Comment:
I think we moved these options around

Reply:
These are the options as in beta 105

Reply:
I think Fraser means moving the options between different groups - happened in #87419 build 15.0.114